src/Data/Fin/Subset/Properties.agda

```diff
@@ -59,6 +67,51 @@ module _ {n} {p q : Subset n} where
   in⊆in-⇔ : p ⊆ q ⇔ inside ∷ p ⊆ inside ∷ q
   in⊆in-⇔ = equivalence in⊆in drop-∷-⊆
 
+  s⊆s : ∀ {s} →  p ⊆ q → s ∷ p ⊆ s ∷ q
+  s⊆s {.inside} p⊆q here = here
+  s⊆s {s} p⊆q (there ∈p) = there (p⊆q ∈p)
+
+  out⊂ : ∀ {y} → p ⊂ q → outside ∷ p ⊂ y ∷ q
+  out⊂ (p⊆q , x , x∈q , x∉p) = out⊆ p⊆q , suc x , there x∈q , λ {(there x∈p) → contradiction x∈p x∉p}
```

Comment:
I think the last part can be simplified to `x∉p ∘ drop-there`

Reply:
Done.